contributors/design-proposals/storage/csi-migration.md

```diff
@@ -35,12 +35,28 @@ lower cost of development as we only have to maintain one version of each
 plugin, as well as ease the transition to CSI when we are able to deprecate the
 internal APIs.
 
+The migration from in-tree plugins to CSI plugins will involve the following phases:
```

Comment:
This should probably have its own section instead of being in the background and motivations. I'm thinking something like "Roadmap" or "Phased Rollout"